comments/__init__.py

```diff
@@ -0,0 +1,13 @@
+import importlib
+
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+
+
+def get_community_moderation_class():
+    try:
+        pkg, attr = settings.COMMENTS_COMMUNITY_MODERATOR_CLASS.rsplit('.', 1)
+        return getattr(importlib.import_module(pkg), attr)
+    except (AttributeError, ValueError):
+        raise ImproperlyConfigured(
```

Comment:
This is breaking change because the app will not work without setting the moderator class correctly first. I would not be able to automatically upgrade existing deployments without changing their configuration first.

Let's set a default class if none is set. The default should be the moderator that always approves. It would be useful to log an informational message to show which class is in use.

Reply:
done